virttest/cpu.py

```diff
@@ -682,6 +682,14 @@ def get_cpu_info(session=None):
     else:
         try:
             output = session.cmd_output(cmd).splitlines()
+            for idx, val in enumerate(output):
+                if re.match(r"^\s+", val):
+                    # Let's adjust unwanted list item
+                    # created for lines which are
+                    # exceeding console line break.
+                    output.remove(val)
+                    output[idx-1] += val.strip(" ")
```

Comment:
My understanding of the fix is to handle the multiline values properly, if so, you can not just strip all the whitespace characters out because some of them are the part of the values. For instance:
```
Vulnerability Spectre v1:        Mitigation; usercopy/swapgs barriers and __user
                                  pointer sanitization
```
the right outcome should be
```
Mitigation; usercopy/swapgs barriers and __user pointer sanitization
```
but by the current approach it would be
```
Mitigation; usercopy/swapgs barriers and __userpointer sanitization
```

Reply:
@luckyh yep, that's right, let me handle that, tnx!